User-Manual.md

```diff
@@ -93,6 +94,15 @@ But don't worry, Tux can find [items](https://github.com/SuperTux/supertux/wiki/
 
 Tux can find powerups, coins and other goodies in [item boxes](https://github.com/SuperTux/supertux/wiki/Items)
 
+### Powerups
+
+Powerups can be found in bonus blocks and on the ground. Most are eggs, which will allow you to backflip by pressing down then jumping and butt-jumping by jumping then pressing down. However, if you already have an egg getting an egg powerup can give you a flower.
```

Comment:
Isn't there supposed to be a comma after "if you already have an egg"? Native English speaker might know more.